### PR TITLE
volumes from

### PR DESCRIPTION
**Pull request by mhald**

### CODE REVIEW

File: docs/packaging-an-application/components-and-containers.md [hunk 1]
```diff
@@ -193,6 +193,31 @@ Optional properties:
       is_excluded_from_backup: true
 ```
 
+We can use "volumes from" to attach several mounts from a colocated container.
+
+```yml:
```
Comment (emosbaugh) [inline docs/packaging-an-application/components-and-containers.md]: im not used to seeing a colon here

File: docs/packaging-an-application/components-and-containers.md [hunk 2]
```diff
@@ -193,6 +193,31 @@ Optional properties:
...
+        action: start
+  - source: public
+    image_name: alpine
+    version: 3
+    ephemeral: true
+    cmd: '["migrate_data.sh"]'
+    volumes_from: ["datastore"]
+```
+
+The container using "volumes from" must start after any containers it mounts from.  Volumes from takes an array of
```
Comment (emosbaugh) [inline docs/packaging-an-application/components-and-containers.md]: `The container using  "volumes_from" must start after any containers it mounts from.  Property "volumes_from" takes...`

**Review by emosbaugh — CHANGES_REQUESTED** [review]
couple of comments

File: docs/packaging-an-application/components-and-containers.md [hunk 3]
```diff
@@ -193,6 +193,31 @@ Optional properties:
...
+        action: start
+  - source: public
+    image_name: alpine
+    version: 3
+    ephemeral: true
+    cmd: '["migrate_data.sh"]'
+    volumes_from: ["datastore"]
+```
+
+The container using "volumes from" must start after any containers it mounts from.  Property "volumes from" takes an array of
```
Comment (emosbaugh) [inline docs/packaging-an-application/components-and-containers.md]: `"volumes from" != "volumes_from"`

**Review by emosbaugh — APPROVED** [review]

File: docs/packaging-an-application/components-and-containers.md [hunk 4]
```diff
@@ -193,6 +193,31 @@ Optional properties:
       is_excluded_from_backup: true
 ```
 
+We can use "volumes_from" to attach several mounts from a colocated container.
```
Comment (marccampbell) [inline docs/packaging-an-application/components-and-containers.md]: Can this become:

"Replicated supports `volumes_from` to attach several mounts from a colocated container."?

**Pull request merged**